src/index.js

```diff
@@ -0,0 +1,204 @@
+import './styles/styles.css';
+
+const refresh = document.querySelector('.uil-sync');
+const input = document.querySelector('input');
+const todosParent = document.querySelector('.todos-container');
+const clearCompletedTaskBtn = document.querySelector('button');
+
+// Object
+class MyObject {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const myArray = [];
```

Comment:
- [ ] Kindly add some example tasks to populate your project
 - 
<img width="960" alt="image" src="https://user-images.githubusercontent.com/61919617/184180991-d793107e-4482-4d18-91eb-a71659d2067c.png">